untitled/src/main/java/com/aca/exam/exam3/entity/UserPlateNumber.java

```diff
@@ -0,0 +1,81 @@
+package com.aca.exam.exam3.entity;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "USERPLATENUMBERS")
+public class UserPlateNumber {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @Column(name = "plate_number", nullable = false)
+    private String plateNumber;
+
+    @ManyToOne
+    @JoinColumn(name = "user_id", foreignKey = @ForeignKey(name = "FK_USER_PLATE_NUMBERS_ID_USERS_ID"), nullable = false)
```

Comment:
the requirement was to generate numbers that do not have owners. The user can not be null in this case

Reply:
it is better to create 3th table to keep user and plate ids there